server/src/controllers/Auth/middleware.ts

```diff
@@ -57,52 +57,6 @@ export type Venues = Merge<
   Prisma.venuesGetPayload<{ select: { id: true; chapter_id: true } }>[]
 >;
 
-// TODO: get events in user middleware and only get those that the user is an
-// event_user for
```

Comment:
@ojeytonwilliams shouldn't this be events belonging to chapters, for which user is in chapter_users? :thinking:

Reply:
Ooops, yes it should.  Technically we do check event roles, but since they don't have any permissions attached to them, they never authorize users!